src/docs/arch-fast-truncate.dox

```diff
@@ -32,8 +32,8 @@ readers.
 The \c page_del field of a \c WT_REF, if not \c NULL, contains information about the
 transaction that deleted the page.
 Its type is \c WT_PAGE_DELETED.
-It can be thought of as a special kind of update, and contains roughly the same
-information.
+The \c page_del can be thought of a special kind of update, as they contain roughly the same
```

Comment:
- you removed "as" before "a special kind" here I think on accident. 
- NP optional: "exact" here is redundant as you already have "same"